src/context/logging.ts

```diff
@@ -174,6 +174,7 @@ interface LogEntry {
 	message: string;
 	log_level: string;
 	error?: string;
+	[key: string]: unknown;
```

Comment:
are the keys associated to the log entry arbitrary bound? there seems to be only two new introductions

```suggestion
	path?: string;
  key_id?: string;
```